rkt/rkt.go

```diff
@@ -100,6 +100,7 @@ func main() {
 		log.SetOutput(os.Stderr)
 	}
 
+	// FIXME(vc): Flags.Args() swallows the "--" which may have stopped its parsing, this interferes with parseApps()
```

Comment:
Probably, it just prevents a graceful exit/error message when someone does something like:
`rkt run -- args for image` where we should error because there's no image yet, but `rkt run` subcommand just gets `args for image`.

I can do a similar check to what I do in parseApps() to see if `--` halted the parsing, and put it back so the subcommand receives it.